src/iperf_client_api.c

```diff
@@ -334,7 +334,7 @@ iperf_handle_message_client(struct iperf_test *test)
     switch (test->state) {
         case PARAM_EXCHANGE:
             if (iperf_exchange_parameters(test) < 0)
-                return -1;
+            return -1;
```

Comment:
Was this removal of leading spaces done by mistake?  Note that iperf3 indentation is 8 spaces.